pkg/consts/consts.go

```diff
@@ -187,6 +187,16 @@ const (
 	BackoffJitterDefault = 1.0
 )
 
+// LB variables for dual-stack
+var (
+	// There's no "service.beta.kubernetes.io/azure-load-balancer" annotation before these 2 new ones are added.
+	// Service.Spec.LoadBalancerIP was used and it will be deprecated.
+	ServiceAnnotationLoadBalancerIPDualStack = map[bool]string{
```

Comment:
Can we make them two strings? It looks strange to have variables in consts file.

Reply:
The intention is to make dualstack code cleaner. Since there will be a lot of `isIPv6 bool` parameters for dualstack support, such map avoids if statements.